### DISCUSSION TITLE
能否固定添加文章Front-matter参数？

### DISCUSSION THREAD

**Discussion by AuntYang** [Qexo/Qexo#326] [General]
我更换 hexo-theme-matery主题后，开新文章每次都要重新配置文章的Front-matter参数，能不能将参数固定在下方，让作者可以选择是否添加参数？亦或者有这个功能，但我并没有在文档中看到相关的配置

**Comment by am-abudu** [comment]
你需要配置博客的模板。参考Hexo文档来添加scaffold

**Reply by AuntYang** [reply]
感谢你的回复，我去看看hexo的文档。
